src/GitPrompt.ps1

```diff
@@ -101,6 +101,8 @@ $global:GitPromptSettings = [pscustomobject]@{
     DefaultPromptDebugSuffix                    = ' [DBG]$(''>'' * ($nestedPromptLevel + 1)) '
     DefaultPromptEnableTiming                   = $false
 
+    AbbreviateHomeDirectory                     = $true

```

Comment:
Can we change this to `DefaultPromptAbbrevHomeDir` or something with the `DeafultPrompt` prefix?